waku/waku_store/resume.nim

```diff
@@ -208,6 +209,7 @@ proc start*(self: StoreResume) {.async.} =
 
     break
 
+  # starting periodic storage of last online timestamp
   self.handle = self.periodicSetLastOnline()
```

Comment:
Wow, never seen like this.... I used asyncSpawn for such cases.

Reply:
I like to think about `AsyncSpawn` as tasks that don't need to be canceled. Having a defined `handle` makes cancelling easy.